src/templates/_index.twig

```diff
@@ -77,7 +77,7 @@
 
                             </span>
 
-                            <span style="padding: 0 2%"> | </span>
+                            <span style="padding: 0 2%">  </span>
```

Comment:
@shnsumit let's remove this entire line.

Reply:
@sidedwards  Done